package.json

```diff
@@ -52,10 +54,11 @@
     "webpack-remove-serviceworker-plugin": "^1.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^29.0.3",
+    "@pinia/testing": "^0.0.15",
+    "@types/jest": "^27.0.1",
     "@types/lodash": "^4.14.185",
     "@types/sinon": "^10.0.13",
-    "@types/vuelidate": "^0.7.15",
+    "@types/vuelidate": "^0.7.16",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
     "@volar-plugins/vetur": "latest",
```

Comment:
I'd recommend upgrading Vue-cli to version 5 - it gets rid of a ton of dependabot vulnerabilities, updates JEST. The version of JEST you're using isn't compatible with Typescript 4.5.5 (the warnings indicate it). Maybe for another ticket?

Reply:
It was decided to upgrade to Vue CLI v5 in a separate ticket. This is what I did for Ketaki's Create UI upgrades and that separate ticket took me the better part of a week. That work is now better understood but I still think it's better to try to keep pieces as small as possible.

Reply:
Travis, just as a reference, the ticket number is: https://github.com/bcgov/entity/issues/16207.